subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/AbstractWorkerExecutorSampleIntegrationTest.groovy

```diff
@@ -41,6 +41,7 @@ abstract class AbstractWorkerExecutorSampleIntegrationTest extends AbstractInteg
     def "creates expected content (#dsl)"() {
         when:
         executer.inDirectory(workerExecutorSample(dsl))
+        executer.withArgument("--stacktrace")
```

Comment:
❓ Is this required or just a leftover from debugging an issue?

Reply:
Yeah, it was just leftover - I've removed it.